lib/letter.rb

```diff
@@ -122,4 +122,32 @@ def update_html(html, annotation)
     end
     return updated
   end
+
+  def strip_html(html)
+    tag_list = %w(
+      a
+      blockquote
+      br
+      div
+      em
+      i
+      img
+      li
+      ol
+      p
+      span
+      ul
+      video
+    )
+
+    tag_list.each { |tag|
+      html.gsub!(/<#{tag}.*?>/i, " ")
+      html.gsub!(/<\/#{tag}>/i, " ")
+    }
+
+    html.gsub!(/[[:space:]]+/, " ")
```

Comment:
I copied it from a Rails method's source code to make sure I wasn't missing anything different in Ruby/Rails. Looking closer now the docs say it is more portable, but every language I've worked with so far treats `\s` like `/[ \t\r\n\f\v]/`, so maybe I should change it to that for simplicity's sake.

_But_ using other POSIX character class syntax like `[[:alnum:]]` and `[[:alpha:]]` catch Unicode characters too, so we should probably use those instead of `\w` of `[a-zA-Z]` going forward. What do you think?

Doc: http://ruby-doc.org/core-2.3.1/doc/regexp_rdoc.html#label-Character+Classes